meson.build

```diff
@@ -0,0 +1,28 @@
+
+project('wgpu-native', 'c', version : '0.6.0', license : 'MPL 2.0')
```

Comment:
are we going to need to bump this version every time we update Cargo.toml?

Reply:
if we provide version information for meson we need it